pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -290,7 +290,9 @@ func (r *ReconcileSyncSetInstance) Reconcile(request reconcile.Request) (reconci
 
 	ssiLog.Info("done reconciling syncsetinstance")
 	if applyErr != nil {
-		return reconcile.Result{}, applyErr
+		ssiLog.Debugf("failed to apply, requeueing: %s", applyErr)
+		// Set requeue to true instead of returning the applyErr to requeue the syncsetinstance.
+		return reconcile.Result{Requeue: true}, nil
```

Comment:
Really glad to learn we can do this and still get backoff.

Reply:
Agree. I am using that behavior elsewhere too. At some point, we should make a push to address how all of our controllers return errors. I suppose that will be needed as part of the upgrade of controller-runtime.